src/pages/order-page/index.tsx

```diff
@@ -1,17 +1,35 @@
-import { useState, Fragment } from 'react'
-import Box from '@mui/material/Box'
+import { useState, Fragment, useEffect } from 'react'
+import { Box, Grid } from '@mui/material'
 import { STEPS } from './constants'
 import { containerStyle } from './style'
 import OrderStepper from '../../components/Order-stepper'
 import StapperButtons from '../../components/Stapper-buttons'
+import OrderContacts from '../../components/Order-contacts'
+import OrderCard from '../../components/Order-card'
+import { getBasketItems } from './operation'
+import { BasketItemsResponse } from '../../types'
 
 const OrderPage = () => {
   const [activeStep, setActiveStep] = useState(0)
+  const [orderList, setOrderList] = useState<BasketItemsResponse[] | null>(null)
```

Comment:
should be empty array as a defaut state, don't use `null`
```suggestion
  const [orderList, setOrderList] = useState<BasketItemsResponse[]>([])
```